ci/e2e/account_roles_files/variables.tf

```
@@ -1,15 +1,21 @@
-variable account_role_prefix {
+variable ocm_environment {
     type = string
+    default = "staging"
 }
 
-variable ocm_env {
+variable openshift_version {
     type = string
-    default = "staging"
+    default = "4.13"
 }
 
+variable account_role_prefix {
+    type = string
+}
 
-variable rosa_openshift_version {
+variable token {
```

for the sake of clarity, let's call it:
```suggestion
variable ocm_token {
```

same goes for ``url``. providing those variables for a user of those files makes it not clear enough what token or url to provide without reading the code